### PR TITLE
updates from wiki before removing wiki

### PR DESCRIPTION
**Pull request by mfdii**
The docs diverged from the wiki. Updating the docs on the site to match the wiki so we can shut down the wiki.

### CODE REVIEW

File: content/docs/rules/_index.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ As noted above, macros provide a way to define common sub-portions of rules in a
 
 With this macro defined, we can then rewrite the above rule's condition as `in_container and proc.name = bash`.
 
-For many more examples of rules and macros, please take a look `rules/falco_rules.yaml`.
```
Comment (lucperkins) [inline content/docs/rules/_index.md]: You can drop the `.html` in link URLs. Not necessary, just FYI.

Comment (mfdii) [reply content/docs/rules/_index.md]: Thanks for the catch. I did that with others, but missed that one.

File: content/docs/event-sources/dropped-events.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Wed Mar 27 15:28:24 2019: Exiting.
+```
+
+## Actions Rate Throttling
+
+To reduce the likelihood of a flood of log messages/alerts, actions are governed by a token bucket. The default parameters of the token bucket are a rate of one alert per 30 seconds with an allowed burst of up to 10 messages.
+
+## Configuration
+
+The actions to take on a dropped syscall event and the throttling parameters for the token bucket are configurable in `falco.yaml` and described [[here|Falco-Configuration#syscall_event_drops]].
```
Comment (lucperkins) [inline content/docs/event-sources/dropped-events.md]: One more thing: this link should be `[here](/docs/configuration/#syscall_event_drops)`. GitHub wiki syntax is unfortunately not supported in standard Markdown.

Comment (mfdii) [reply content/docs/event-sources/dropped-events.md]: Nice catch. Just missed updating it.

**Pull request merged**